src/uwtools/resources/jsonschema/fv3.jsonschema

```diff
@@ -141,10 +99,10 @@
           "type": "object"
         },
         "files_to_copy": {
-          "$ref": "#/$defs/filesToStage"
+          "$ref": "urn:uwtools:files-to-stage"
```

Comment:
This is how a reference to another schemas now looks.

Reply:
Could you explain what `urn` is for? I was looking this up in the JSONSchema documentation and didn't find a hit minus this is how you reference an external schema. Assuming the `utwools:files-to-stage` part of `urn:uwtools:files-to-stage` shows a path of where this schema now lives based on the new file created within resources/jsonschema

Reply:
As background, the `jsonschema` package now handles inter-schema references via the [`referencing`](https://referencing.readthedocs.io/en/stable/intro/) library, and has provided [a guide](https://python-jsonschema.readthedocs.io/en/latest/referencing/#migrating-from-refresolver) for migrating away from the previous mechanism.

URNs are conceptually related to URLs and URIs -- see e.g. https://pitt.libanswers.com/faq/213722. They are simple, and defined by [RFC 2141](https://www.ietf.org/rfc/rfc2141.txt). The only really interesting thing about them for `uwtools` is that `referencing` -- and, thus, `jsonschema` -- recognized and accepts them as the values of `$ref` values in schema documents. (Search for `urn:` in [this page](https://referencing.readthedocs.io/en/stable/intro/) to see use cases from the `referencing` docs.) Our code takes care of mapping these URN values to schema files, loading those, and adding them to a "registry" object that  `jsonschema` then uses to resolve the `$ref`s.

Our other option for `$ref` values would be to use a URI scheme. You can see lots of examples of this on the internet, where people might use, in a case like the one you cite above, like `https://uwtools.com/files-to-stage`. I find this uncomfortable because, as I understand it, it really has nothing to do with HTTPS and no network access is done to resolve the references. All the URI scheme is really providing here is _hierarchy_ -- a way to identify resources based on the slash characters separating components. The `uwtools.com` domain doesn't have to exist, and could probably be any text as long as it agreed with other URIs, at least in some cases. Alternatively, we could use something like `file://uwtools/files-to-stage`, but I find this misleading, too. I think that any URI scheme (and there are [lots of official ones](https://referencing.readthedocs.io/en/stable/intro/)) would do.

I find the URNs conceptually cleaner. There is a potential downside in that, as far as I can tell, URNs don't provide a way to "drill down" into a referenced schema the way URIs can. For example, if I had a big `common.jsonschema` document with lots of schemas, I might write `file://uwtools/common#/properties/namelist` to reference a sub-schema in that document. I think with URNs we'll need to have a one-to-one mapping between URNs a schema files, but that might be ok.